.github/CODEOWNERS

```diff
@@ -4,9 +4,7 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/api-logging is the default owner for changes in this repo
-*                       @googleapis/yoshi-java @googleapis/api-logging
-**/*.java               @googleapis/api-logging
+*                       @googleapis/api-logging @googleapis/yoshi-java
 
 # The java-samples-reviewers team is the default owner for samples changes
-samples/**/*.java       @googleapis/java-samples-reviewers
+samples/**/*.java       @googleapis/java-samples-reviewers @googleapis/api-logging
```

Comment:
This was intentional to ensure that samples met the Java samples styleguide. We can work on adding folks from the observability team to the samples reviewers group.

Reply:
Reverting this change. It would be useful to get comms on what's happening on the yoshi side - it's not the first time that we discover surprise changes in the repos we own without prior communication.